CMakeLists.txt

```diff
@@ -2,39 +2,51 @@ cmake_minimum_required(VERSION 3.20)
 
 project(libkmeans LANGUAGES Fortran)
 
-# Build directory
-set(CMAKE_Fortran_BIN_DIRECTORY ${PROJECT_BINARY_DIR}/bin)
-set(CMAKE_Fortran_LIB_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
 set(CMAKE_Fortran_MODULE_DIRECTORY ${PROJECT_BINARY_DIR}/modules)
 
 # Compiler flags for GCC only
+# Note, flags "should" be specified in yaml presets, and passed to cmake as arguments
+# Use c-preprocessor. Note, probably better to set target property `Fortran_PREPROCESS` instead
```

Comment:
```suggestion
# Use c-preprocessor. Note, probably better to set target property `Fortran_PREPROCESS` instead
```

You don't need to do anything, it is automatic. Just rename the files `.f90` to `.F90`